ui/src/components/applications/Applications.vue

```diff
@@ -445,29 +445,6 @@ export default {
         context.initGraphics();
         window.top.document.title = name + " - " + hostname;
       }, 2000);
-    },
-    getHideUninstall() {
```

Comment:
we have to keep this part, it is used to show the dotted menu

Reply:
the function getHideUninstall is back